### PR TITLE
feat() report back to /ws menu navigation

### PR DESCRIPTION
**Pull request by asturur**
This PR adds a new WS kind of message to inform the browser of the current menu item.
The messages are in the format:

menuNavigation:folder:menuItem

image

it works for console and files too

image

The timing is pretty much ok, between 20 and 25ms pass between pressing UP on the remote and getting the message back on the remote

### CODE REVIEW

File: cmd/remote/games/tracker.go [hunk 1]
```diff
@@ -3,14 +3,15 @@ package games
...
+	"path/filepath"
+
 	"github.com/wizzomafizzo/mrext/cmd/remote/websocket"
 	"github.com/wizzomafizzo/mrext/pkg/config"
 	"github.com/wizzomafizzo/mrext/pkg/mister"
 	"github.com/wizzomafizzo/mrext/pkg/service"
 	"github.com/wizzomafizzo/mrext/pkg/tracker"
-	"net/http"
-	"os"
-	"path/filepath"
```
Comment (asturur) [inline cmd/remote/games/tracker.go:13]: this is just VSCODE having opinions on import order. If i find the option i can disable it

File: docs/remote-api.md [hunk 2]
```diff
@@ -1648,6 +1647,15 @@ Format: `gameRunning:{system}/{name}`
...
 This field is blank if no game is running.
 
+#### Menu status
+
+Format: `menu:{system}/{name}`
+
+| Attribute | Type   | Description |
+|-----------|---------|------------------|
+| `system`  | string | System ID of currently running core. |
+| `name`    | string | Filename of currently running game. |
```
Comment (asturur) [inline docs/remote-api.md:1657]: TBD

File: pkg/tracker/tracker.go [hunk 3]
```diff
@@ -247,7 +249,9 @@ func (tr *Tracker) addEvent(action int, target string) {
 		ev.TargetPath = targetTime.Path
 	}
 
-	tr.Events = append(tr.Events, ev)
+	if action != EventActionMenuNavigation {
+		tr.Events = append(tr.Events, ev)
+	}
```
Comment (asturur) [inline pkg/tracker/tracker.go:254]: i don't want to append here all the navigation events, seems too much

File: cmd/remote/games/tracker.go (hunk 1 above)
Comment (wizzomafizzo) [reply cmd/remote/games/tracker.go:13]: No problem. I use vs code now so you can leave it. It was a Goland thing before

**Pull request merged**